srcpkgs/ipp-usb/files/71-ipp-usb.rules

```diff
@@ -0,0 +1 @@
+ACTION=="add", SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ENV{ID_USB_INTERFACES}=="*:070104:*", OWNER="root", GROUP="lp", MODE="0664", RUN+="/usr/bin/sv o ipp-usb"
```

Comment:
Why can't we use https://github.com/OpenPrinting/ipp-usb/blob/master/systemd-udev/71-ipp-usb.rules ?

Reply:
runit

Reply:
Added a note to force running the service once on boot.

Reply:
Should I add relevant oneshot services/crontab examples?

Reply:
@oynqr please make the changes to the udev rule a patch to https://github.com/OpenPrinting/ipp-usb/blob/master/systemd-udev/71-ipp-usb.rules , so we can adapt to changes they make instead of having to remember to check.

Once that's done should be good to merge.